### ISSUE TITLE
Arguments mismatch for follow: "value" with value "all"

### ISSUE THREAD

**Issue by jiripospisil** [laino/final-pm#9]
Hello @laino! It looks like there's an issue when trying to `log -f` all of the available apps. I assume `all` should be supported here as well. If not, please consider this a feature request :).

```
final-pm log -f all
[ERROR] [CLI] Arguments mismatch for follow: "value" with value "all" matches the inverted is not "all" pattern
```

**Comment by laino** [comment]
**tl;dr:** This is implemented in ``2.24.0``.

**Long Explanation:**

In the daemon ``follow``/``unfollow`` "all" is implemented via a virtual application (really just an event) that receives all logging output. The call just wasn't whitelisted.

This was because it isn't as transparent as I would like: for instance you can't ``follow`` "all", then ``unfollow`` some applications separately to get "all except X, Y and Z". Also following "all" would different to other commands in that it doesn't really operate on some pre-defined set of applications/processes, since obviously you'd also get log output of applications that are added later.

There may also be some counter-intuitive behavior if you ``follow`` "all", but ``follow`` more applications separately (duplicate log lines instead of NOOP), but the CLI already detects and prevents this.

Plus (!) "all" is generally resolved to a list of known applications on the client side (to prevent ugly surprises
where something your client didn't see earlier gets caught in it) and I had originally planned to keep the concept of "all" out of the daemon entirely, where I had to relent when it came to logging. At least here it's safe to allow it

I did not really consider how counter-intuitive this decision would make the CLI client, where ``unfollow`` doesn't matter anyways and following everything, even if it is added later, is probably precisely what you meant to do. Also the CLI documentation nowhere states that "all" is unsupported for ``-f``, so that would still need changing.

I'm now convinced it is better to just allow "all" in follow/unfollow, which is implemented in version ``2.24.0`` by whitelisting the argument.

**Comment by jiripospisil** [comment]
Thank you! Works great.
